lib/presentation/atoms/progress_indicator_atom.dart

```diff
@@ -44,8 +46,23 @@ class _ProgressIndicatorAtomState extends State<ProgressIndicatorAtom>
 
   @override
   Widget build(BuildContext context) {
-    return LinearProgressIndicator(
-      value: controller.value,
+    final ThemeData themeData = Theme.of(context);
+    final ColorScheme colorScheme = themeData.colorScheme;
+    return SizedBox(
+      width: double.infinity,
+      height: 35,
+      child: LiquidLinearProgressIndicator(
+        value: controller.value,
+        backgroundColor: colorScheme.outline,
+        valueColor: AlwaysStoppedAnimation<Color>(colorScheme.primary),
+        borderColor: colorScheme.outline,
+        borderRadius: 10,
```

Comment:
We should make the border-radius same for all the app so it should move to GlobalVariables.

Please change it in the second pr where you switch ProgressIndicatorAtom percentage to being controlled from outside.

Reply:
Since the PR is already merged, do I make the changes and reopen a PR?